mbf_abstract_nav/src/abstract_controller_execution.cpp

```diff
@@ -185,6 +186,47 @@ void AbstractControllerExecution::setVelocityCmd(const geometry_msgs::TwistStamp
   // TODO so there should be no loss of information in the feedback stream
 }
 
+bool AbstractControllerExecution::checkCmdVelIgnored(const geometry_msgs::Twist& cmd_vel)
+{
+  // check if the velocity ignore check is enabled or not
+  if (cmd_vel_ignored_tolerance_ <= 0.0)
+  {
+    return false;
+  }
+
+  const bool robot_stopped = robot_info_.isRobotStopped(1e-3, 1e-3);
+
+  // compute linear and angular velocity magnitude
+  const double cmd_linear = std::hypot(cmd_vel.linear.x, cmd_vel.linear.y);
+  const double cmd_angular = std::abs(cmd_vel.angular.z);
+
+  const bool cmd_is_zero = (cmd_linear < 0.01 && cmd_angular < 0.01);
```

Comment:
S: maybe use 1e-3 here too?  
S: I think u can remove the brackets (not 100% sure)

Reply:
I tested without the brackets and it was working fine. So I removed it.